webrepl_cli.py

```diff
@@ -36,13 +36,14 @@ def __init__(self, s):
             self.s = s
             self.buf = b""
 
-        def write(self, data):
+        def write(self, data, text=False):
+            OpCode = 1 if text else 2
```

Comment:
Codestyle: you violate both PEP8 and convention of this file (which of course uses standard_python_style).